packages/bitcore-wallet-client/lib/api.js

```diff
@@ -459,9 +459,14 @@ API.prototype._import = function(cb) {
   });
 };
 
+
+API.prototype.importAllFromMnemonic = function(words, opts, cb) {
```

Comment:
what is this, is it going to be used later?

Reply:
removed for now.